manifests/07_operator.yaml

```diff
@@ -3,6 +3,8 @@ kind: Deployment
 metadata:
   name: marketplace-operator
   namespace: openshift-marketplace
+  annotations:
+    config.openshift.io/inject-proxy: "marketplace-operator"
```

Comment:
is the value arbitrary?

Reply:
Based on [this line in the PR](https://github.com/openshift/cluster-version-operator/pull/224/files#diff-20dbece9affd4bfefb9de7caca6bc037R52) it looks like you need to provide the names of the pods.